### ISSUE TITLE
yate qt5 compatibility?

### ISSUE THREAD

**Issue by jmrcpn** [yatevoip/yate#2]
TRying to include yate within osukiss distribution.
Yate seems not able to compile while Qt (version 5.15.7) is present (problem finding include directory QSound).
Tried to "fine tune" configure.ac , but  qt4client.cpp is not qt5 compatible.
Is there way to use yate with qt5?  plan to do it?

**Comment by mnhauke** [comment]
All major Linux distributions dropped support for Qt4 years ago.
Since then they had to build their yate packages without the parts that depend on Qt4.

Gentoo
https://bugs.gentoo.org/642544
https://gitweb.gentoo.org/repo/gentoo.git/commit/net-voip/yate?id=c4e19115458f5b8f8d1294c02d5dcdb3c2ab7049
https://bugs.gentoo.org/776340

Arch
https://gitlab.archlinux.org/archlinux/packaging/packages/yate/-/commit/97786d666587e32551142987a2ae8e5cd60665dd

SUSE
https://build.opensuse.org/request/show/854590/changes
https://bugzilla.opensuse.org/show_bug.cgi?id=1179613

It would be nice to have the old code ported over to Qt5 / Qt6.

Here's the info from the old bug tracking system (even with some experimental patches to support Qt5)
https://web.archive.org/web/20220819112150/https://yate.null.ro/mantis/view.php?id=421

**Comment by svedm** [comment]
If it's critical you can use changes from my [old fork](https://github.com/svedm/yate). I will try to rebase it and create PR to this repo on weekends.

**Comment by laf0rge** [comment]
See alos the previous mantis bug report at https://web.archive.org/web/20220819112150/https://yate.null.ro/mantis/view.php?id=421

**Comment by alexander-pick** [comment]
If you don't need yate-qt4, just add `--without-libqt4` to configure and it will build.
